projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.ts

```diff
@@ -45,12 +45,20 @@ export class DefaultHoldingItemComponent implements OnInit {
   /** Item permissions */
   permissions: any;
 
+  // GETTER & SETTER ==========================================================
   /** Current interface language */
   get language() {
     return this._translateService.currentLang;
   }
 
-  // CONSTRUCTOR & HOOKS ==============================================================
+  /** Get the number of pending request on this item */
+  get itemRequestCounter() {
+    return (this.item.metadata.availability && this.item.metadata.availability.request)
+      ? this.item.metadata.availability.request
```

Comment:
What is `request` an array an integer?

Reply:
`metadata.availability` is an object regrouping all necessary information to display (as well as possible) the item availability :
```json
{
  "available": false,
  "display_text": [
    {
      "label": "on_shelf",
      "language": "default"
    }
  ],
  "request": 1,
  "status": "on_shelf"
}
```

These informations are dumped from backend serialize.
So the `request` field is the number of pending request related to the item. It is always present into object.